packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -105,14 +106,33 @@ void main() {
         dirToDelete.deleteSync(recursive: true);
       }
     }
+    Timer timeout = new Timer(_kPlatformTimeout, () {
+      process.stderr
+        .transform(UTF8.decoder)
+        .transform(const LineSplitter())
+        .listen((String line) {
+          if (line != null)
+            print(line);
+        });
+      finalize();
+      throw new TimeoutException("Test took too long", _kPlatformTimeout);
```

Comment:
Maybe "Test timed out. This may indicate a syntax error in the test (try using "flutter analyze" on the test). If an exception occurred, it will have been printed above this message."

(Or below this message, I forget in what order they print out.)